bb8/src/internals.rs

```diff
@@ -22,6 +22,27 @@ where
     pub(crate) statistics: AtomicStatistics,
 }
 
+pub(crate) struct GetGuard<M: ManageConnection + Send> {
+    inner: Arc<SharedPool<M>>,
+}
+
+impl<M: ManageConnection + Send> GetGuard<M> {
+    fn new(inner: Arc<SharedPool<M>>) -> Self {
+        {
+            let mut locked = inner.internals.lock();
+            locked.inflight_gets += 1;
+        }
```

Comment:
Nit: there's not really any point in this scope, right?

Reply:
Fair, I can remove it

Reply:
Ah sorry, I forgot I had this because I need to drop the MutexGuard before I can create the GetGuard.

```
cannot move out of `inner` because it is borrowed
```

Adding the scope felt better than doing a `.clone()`